src/acceptor.rs

```diff
@@ -0,0 +1,139 @@
+use core::task::{Context, Poll};
+use std::future::Future;
+use std::io;
+use std::pin::Pin;
+use std::sync::Arc;
+
+use futures_util::ready;
```

Comment:
It looks like `ready()` isn't actually being used, so maybe we don't need this? (In which case we can lose the new dependency and thus the Cargo feature.)

Reply:
`ready!()` is used in a few places in the code below